docs/how-to-create-a-new-release.rst

```diff
@@ -103,27 +103,25 @@ still work on a new installation. This is important because it can find missing
 Building the documentation
 --------------------------
 
-An important part of the release process is ensuring that the documentation for readthedocs_ site can be built. This can
-be tested locally by executing the following commands in the repository folder::
+Well documented code is an essential part of the release, allowing your code's legacy to only grow in glory and admiration.
+The documentation will be rendered via the readthedocs_ site, allowing future developers, practitioners, researchers and students
+to understand and build upon your work. Note: entry points created by the command ``cea install-toolbox`` should be run at least once
+and you also need to install GraphViz_ to produce the graphs.
 
-    cd docs
-    make clean
-    make html
+CEA uses sphinx_ to document all module code, and has a tool specific to documentation to make your life a little easier. Simply run via the command
+line once the cea virtual environment is activated in conda: ``activate cea``::
```

Comment:
instead of ``activate cea``, just launch the CEA Console installed by the Windows Installer - it already has all the paths and stuff set. Documentation should start with "Launch the CEA Console" and continue from there...

Reply:
sure! i didn't realise we had a dedicated console now, that's pretty cool

Reply:
has all the documentation been updated to reflect this?